docs/security/unauthorized-sign-in.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Unauthorized sign-in
+description: Notify users of unauthorized sign-ins to their accounts
+---
+
+Clerk identifies attempts by unauthorized individuals or entities to access a user's account.
+
+## Detecting unauthorized sign-ins
+
+Whenever a sign-in attempt is completed from an unfamiliar device, Clerk sends an email notification to the account owner with information about the newly created session.
+
+The account owner can then choose to immediately revoke this session.
+
+To configure this feature:
+
+1. In the Clerk Dashboard, navigate to the [**Emails**](https://dashboard.clerk.com/last-active?path=customization/email) page.
+1. Select the **New Client Sign In** option. You'll be redirected to the template settings page.
+1. Toggle **Delivered by Clerk** to enable the feature.
+1. Select **Apply changes**.
+1. To customize the email content, see the [Email and SMS templates](/docs/authentication/configuration/email-sms-templates) guide.
+
+## Email notification for unauthorized access
+
+The email notification users receive for unauthorized sign-ins varies depending on the instance's configuration and the application's billing plan.
+
+By default, the email includes information about the unauthorized sign-in attempt, such as device type, operating system, IP address, location, and the sign-in method used. If you've set a support email for your app, Clerk will add instructions for the user to contact the app administrator.
+
+For supported instances, the email might also include a button allowing users to sign out from the unrecognized device. Clicking this button immediately revokes the session.
+
+Customize the unauthorized sign-in email notification:
+
+1. In the Clerk Dashboard, navigate to the [**Emails**](https://dashboard.clerk.com/last-active?path=customization/email) page.
+1. Select the **New Client Sign In** option. You'll be redirected to the template settings page.
+1. Edit the email template and apply your changes.
+
+## Session revocation for unauthorized sign-ins
+
+> [!WARNING]
+> This feature isn't available in production for free plans. However, you can try it out free in development mode. For more information, see the [pricing](/pricing){{ target: '_blank' }} page.
+
+For apps that support it, users can immediately revoke unauthorized sign-ins directly from the email notification. A single click will revoke the suspicious session and redirect the user to a confirmation page. The contents of the confirmation page depend on the instance's configuration.
```

Comment:
```suggestion
For supported instances, the email might also include a button allowing users to immediately revoke unauthorized sign-ins. Clicking this button revokes the suspicious session and redirects the user to a confirmation page. The contents of the confirmation page depend on the instance's configuration.
```

Reply:
is this configurable? it says "for supported instances", so what instances are supported? when will the email include the button to revoke the session?

Reply:
The "supported instances" are either development instances or production instances for an application on a paid add-on.